adoc/chapters/programming_interface.adoc

```diff
@@ -12349,6 +12349,17 @@ implementation-defined object of unspecified type, which is interpreted by
 [code]#parallel_for# to construct an appropriate [code]#reducer#
 type as detailed in <<reducer-class>>.
 
+If an identity value is supplied to a reduction, an implementation may use that
+value to initialize an unspecified number of temporary variables inside of any
+[code]#reducer# objects it creates.
+
+[NOTE]
+====
+Since the number of temporary variables is unspecified, supplying an identity
+value different to the identity value associated with the reduction operator
+may lead to unexpected results.
+====
+
 The initial value of the reduction variable is included
 in the reduction operation, unless the [code]#property::reduction::initialize_to_identity#
 property was specified when the [code]#reduction# interface was invoked.
```

Comment:
Do we need a comment to say the identity will be the known one, unless you specify one and then it will be the one you specified?

Reply:
I agree this needed to be clarified.  Rather than add something here, I've tried to clarify the expected behavior in the description of `initialize_to_identity` property itself.  How does 41cb15937975d2ccae97bcced63eb679c76ad054 look?

Reply:
Yep, that looks great.